model/trace/instrumentation/graphql.yml

```diff
@@ -1,6 +1,6 @@
 groups:
-  - id: graphql
-    prefix: graphql
+  - id: graphql.server.request
```

Comment:
This are just the `graphql.server` spans. Should we also specific the `graphql.client` spans?

Reply:
If it is desired, yes we should. But most likely not as part of this PR. Also, maybe it's good to create an issue so maybe discussions can happen there.